simple/doxbee-promise.js

```diff
@@ -105,14 +105,24 @@ module.exports = function doxbee(stream, idOrPath) {
     });
 };
 
+module.exports = {
+  doxbee,
+  fakes,
+};
+
 },{"../lib/fakes-promises.js":2}],2:[function(require,module,exports){
 "use strict";
 
-function dummy_1() { return Promise.resolve(undefined); }
-function dummy_2(a) { return Promise.resolve(undefined); }
-
 // a queryish object with all kinds of functions
 function Queryish() {}
+Queryish.count = 0;
+async function dummy_1() {
+  Queryish.count++;
+}
+async function dummy_2(a) { 
+  Queryish.count++;
+}
```

Comment:
They should avoid using `async`, and do
```
function dummy_1() {
  Queryish.count++;
  return Promise.resolve(undefined);
}

function dummy_2(a) { 
  Queryish.count++;
  return Promise.resolve(undefined);
}
```
because that's the purpose of this `doxbee-promise` (not async, that's tested in `doxbee-async`. testing chain of promises).

Reply:
I'm still observing quite a diff even with this fixed for JSC (almost none for V8):

Before: `321.0 ± 11.0(3.35%)`
After: `309.0 ± 13.0(4.24%)`